commands/volume.js

```diff
@@ -25,8 +25,13 @@ module.exports = {
         if (message.guild.me.voice.channel && message.member.voice.channel.id !== message.guild.me.voice.channel.id) return client.sendTime(message.channel, ":x: | **You must be in the same voice channel as me to use this command!**");
         if (!parseInt(args[0])) return client.sendTime(message.channel, `**Please choose a number between** \`1 - 100\``);
         let vol = parseInt(args[0]);
+        if(vol>0 && vol>101){
```

Comment:
```suggestion
        if(vol < 0 || vol > 100){
```

Reply:
Check this code if its work let me know so i can commit

Reply:
yes works either way ;-;